grid/recruitment/get-cv/maps/get-cv.ts

```diff
@@ -0,0 +1,112 @@
+/* eslint-disable jest/no-export */
+
+import { RecordingProcessOptions, SuperfaceTest } from '@superfaceai/testing';
+
+import { buildSuperfaceTest } from '../../../test-config';
+
+export const getCVTest = (
+  provider: string,
+  testInputs: {
+    validCandidateId: string;
+    invalidCandidateIds: string[];
+  },
+  options?: RecordingProcessOptions
+): void => {
+  describe(`recruitment/get-cv/${provider}`, () => {
+    let superface: SuperfaceTest;
+
+    describe('GetCV', () => {
+      beforeAll(() => {
+        superface = buildSuperfaceTest({
+          profile: 'recruitment/get-cv',
+          provider,
+          useCase: 'GetCV',
+        });
+      });
+
+      describe('when geting CV with valid candidate ID', () => {
+        it('returns CV document', async () => {
+          const result = await superface.run(
+            {
+              input: {
+                candidateId: testInputs.validCandidateId,
+              },
+              testName: 'get CV with valid candidate ID',
+            },
+            options
+          );
+
+          expect(() => result.unwrap()).not.toThrow();
+          expect(result).toMatchSnapshot();
+        });
+      });
+
+      testInputs.invalidCandidateIds.forEach(invalidCandidateId => {
+        describe(`when geting CV with invalid candidate ID ${invalidCandidateId}`, () => {
+          it('returns candidate not found error', async () => {
+            const result = await superface.run(
+              {
+                input: {
+                  candidateId: invalidCandidateId,
+                },
+                testName: `get CV with invalid candidate ID ${invalidCandidateId}`,
+              },
+              options
+            );
+
+            expect(() => result.unwrap()).toThrow();
+            expect(result).toMatchSnapshot();
```

Comment:
sanity check - this ensures it really returns `candidate not found` error and not a different error, right?

Reply:
yes, the snapshot validation ensures it